blog-service/2023-01-30-slo-monitors.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Monitor-Based SLOs
+image: https://www.sumologic.com/img/logo.svg
+keywords:
```

Comment:
should we also add `slos` as a keyword ?

Reply:
Sure, I can push this out with the next round of edits